contracts/CallbackGnosis.sol

```diff
@@ -0,0 +1,83 @@
+pragma solidity ^0.8.0;
+import "./interfaces/ICallbackGnosis.sol";
+
+/// @notice Operated via the GnosisFactory's createProxyWithCallback method
+/// @notice Purpose - Setup Gnosis Safe w/ any arb method calls
+contract CallbackGnosis is ICallbackGnosis {
+    /// @dev Method called once a proxy contract is created
+    /// @param proxy GnosisSafe address
+    /// @param _singleton GnosisSafe impl address
+    /// @param initializer Payload used to setup GnosisSafe Configuration 
+    /// @param saltNonce Salt utilized for GnosisSafe Create2 opcode
+    function proxyCreated(
+        GnosisSafeProxy proxy,
+        address _singleton,
+        bytes calldata initializer,
+        uint256 saltNonce
+    ) external {
```

Comment:
Since the _singleton and saltNonce variables aren't used here, you can remove the variable names while still satisfying the IProxyCreationCallback interface. This will clear those linter warnings.

```
    /// @dev Method called once a proxy contract is created
    /// @param proxy GnosisSafe address
    /// @param initializer Payload used to setup GnosisSafe Configuration 
    function proxyCreated(
        GnosisSafeProxy proxy,
        address,
        bytes calldata initializer,
        uint256
    ) external {
```